tests/session/session-endSession-001.phpt

```diff
@@ -0,0 +1,77 @@
+--TEST--
+MongoDB\Driver\Session::endSession() Calling methods after session has been ended
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_clean(DATABASE_NAME, COLLECTION_NAME); ?>
+<?php skip_if_not_replica_set(); ?>
+<?php skip_if_server_version('<', '4.0'); ?>
```

Comment:
Since we're not actually executing transactions here, I think you can relax these skips:

```
<?php skip_if_not_live(); ?>
<?php skip_if_server_version('<', '3.6'); ?>
```

The above is taken from the original session tests.

Reply:
Unless this relates to you testing the script without `$sessionA->endSession();`?

Reply:
It doesn't need the ``if_not_clean`` I guess, but all the others should stay. I'm not relaxing it.